brian2/codegen/generators/cython_generator.py

```diff
@@ -546,7 +548,7 @@ def determine_keywords(self):
                                                              name='_clip')
 
 timestep_code = '''
-cdef int _timestep(double t, double dt):
+cdef int64_t _timestep(double t, double dt):
```

Comment:
Any reason we use ``int64_t`` in some places and ``size_t`` in others? Actually it's just here - because ``_timestep`` should always have been 64-bit I guess?

Reply:
Right, I fixed this along the way, this should have been `int64_t` for a while. It is somewhat unrelated to the other changes.